student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Student < Person
+  def initialize(classroom)
+    @classroom = classroom
+    super
```

Comment:
- [ ] Your initialize method does not call the super method with all the required methods. Please try to refer to the example below to complete yours 👇 
![image](https://user-images.githubusercontent.com/90222110/193605498-6db28cff-aaa6-4fce-9bcf-885685879acc.png)